### PR TITLE
added unit test for all the classes of presenter

### PR DESCRIPTION
**Pull request by LucreciaLucero**
+ **Unit test with coverage**

image
image

+ **Structure Folder**

image

### CODE REVIEW

File: app/src/test/java/com/example/parkingsystemjava/mvp/presenter/DialogFragmentPresenterTest.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    private final static String PARKING_LOT_EMPTY_STRING = "";
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.openMocks(this);
+        presenter = new DialogFragmentPresenter(view);
+    }
+
+    @Test
+    public void onSaveButtonPressedTest() {//el toast lo tengo en el view de la actividad ?? - no lo puedo llamar aca
```
Comment (SartiMau) [inline app/src/test/java/com/example/parkingsystemjava/mvp/presenter/DialogFragmentPresenterTest.java]: remove this comment pls and if it is necessary just put it in a new line and in english

File: app/src/test/java/com/example/parkingsystemjava/mvp/presenter/DialogFragmentPresenterTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        when(view.getParkingsAvailable()).thenReturn(PARKING_LOT_STRING);
+
+        presenter.onSaveButtonPressed(dialogFragmentListener);
+
+        verify(view).dismissDialogFragmentParkingLots();
+    }
+
+    @Test
+    public void onSaveButtonPressedEmptyStringTest() {//el toast lo tengo en el view de la actividad ??
```
Comment (SartiMau) [inline app/src/test/java/com/example/parkingsystemjava/mvp/presenter/DialogFragmentPresenterTest.java]: same

**Review by SartiMau — CHANGES_REQUESTED** [review]
Please undo the mvp package rename

File: app/src/test/java/com/example/parkingsystemjava/mvp/presenter/DialogFragmentPresenterTest.java (hunk 2 above)
Comment (LucreciaLucero) [reply app/src/test/java/com/example/parkingsystemjava/mvp/presenter/DialogFragmentPresenterTest.java]: Done

File: app/src/test/java/com/example/parkingsystemjava/mvp/presenter/DialogFragmentPresenterTest.java (hunk 1 above)
Comment (LucreciaLucero) [reply app/src/test/java/com/example/parkingsystemjava/mvp/presenter/DialogFragmentPresenterTest.java]: Done

**Review by SartiMau — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

**Review by nicolbidone — APPROVED** [review]

**Pull request merged**